src/frontend/servers/y-provider/jest.config.js

```diff
@@ -1,4 +1,4 @@
-var config = {
+const config = {
```

Comment:
This can be enforced by the use of this eslint rule BTW:
https://eslint.org/docs/latest/rules/no-var

Reply:
Add a commit that modify configuration to include root file.